images/linux/scripts/helpers/install.sh

```diff
@@ -95,3 +95,68 @@ get_github_package_download_url() {
     fi
     echo $downloadUrl
 }
+
+get_github_package_hash() {
+    local repo_owner=$1
+    local repo_name=$2
+    local file_name=$3
+    local url=$4
+    local version=${5:-"latest"}
+    local prerelease=${6:-false}
+    local delimiter=${8:-'|'}
+    local word_number=${9:-2}
```

Comment:
Similar PowerShell function has default value `1` here
```suggestion
    local word_number=${9:-1}
```

Reply:
I spent about an hour to find that PS start counting from 0 but Bash from 1, so let it be 2 to be the same as in PS. 🤣